### PR TITLE
Updated Japanese

### PR DESCRIPTION
**Pull request by dabura667**

### CODE REVIEW

**Comment by cmgustavo** [comment]
Please, needs rebase.

**Comment by dabura667** [comment]
rebased

File: po/ja.po [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
 msgstr ""
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
 "Project-Id-Version: Copay\n"
-"Plural-Forms: nplurals=1; plural=0;\n"
 "POT-Creation-Date: \n"
 "PO-Revision-Date: \n"
 "Last-Translator: \n"
 "Language-Team: dabura667\n"
+"Language: ja_JP\n"
```
Comment (cmgustavo) [inline po/ja.po]: Could you use generic language ISO-639-1? I mean, not country specific, just the language. 
In your case, just `Language: ja`. We are not supporting localized languages yet.

**Comment by matiu** [comment]
Sorry, I closed this one by error.

**Comment by dabura667** [comment]
fixed to ja

**Comment by cmgustavo** [comment]
Thanks!

**Pull request merged**